bin/syclcc-clang

```diff
@@ -545,33 +605,26 @@ class hip_invocation:
     self._rocm_path = config.rocm_path
     self._clang = config.clang_path
     self._linker_args = config.rocm_link_line
+    self._cxx_flags = config.rocm_cxx_flags
     self._clang_include_path = config.clang_include_path
     self._hipsycl_lib_path = os.path.join(config.hipsycl_installation_path, "lib/")
 
   def get_compiler_preference(self):
     return (self._clang, 100)
 
   def get_cxx_flags(self):
-
-    flags = [
+    flags = self._cxx_flags
+    flags += [
         "-x", "hip",
-        "--hip-device-lib-path=" + os.path.join(self._rocm_path, "lib"),
         "-D__HIPSYCL_ENABLE_HIP_TARGET__",
         "-D__HIPSYCL_CLANG__",
-        # clang erroneously sets feature detection flags for 
-        # __float128 even though it is not supported for CUDA / HIP,
-        # see https://bugs.llvm.org/show_bug.cgi?id=47559.
-        "-U__FLOAT128__", "-U__SIZEOF_FLOAT128__",
-        "-I" + os.path.join(self._rocm_path, "include"),
         # This here is necessary because some distributions of clang
         # incorrectly have the gcc/system headers *before* clangs own
         # include path. This causes "file not found" errors when the
         # clang CUDA wrapper headers try to include the system headers
         # with #include_next <...>
```

Comment:
This comment refers to the `-isystem` that is now absorbed into the cxx flags, so it can go, right?